src/sif/autocost.cc

```diff
@@ -76,13 +76,6 @@ class AutoCost : public DynamicCost {
   virtual bool AllowMultiPass() const;
 
   /**
-   * Disables entrance into destination only areas. This should only be used
```

Comment:
Removed override of DisableDestinationOnly. The base-class function is sufficient.